osi_common.proto

```diff
@@ -579,3 +580,57 @@ message WavelengthData
     //
     optional double samples_number = 3;
 }
+
+//
+// \brief A description for the positions of the pedals.
+//
+message Pedalry
+{
+    // Position of the acceleration-pedal.
+    // Range: 0-1 (Unpressed - fully pressed)
+    //
+    optional double pedal_position_acceleration = 1;
+
+    // Position of the brake-pedal.
+    // Range: 0-1 (Unpressed - fully pressed)
+    //
+    optional double pedal_position_brake = 2;
+
+    // Position of the clutch-pedal.
+    // Range: 0-1 (Unpressed - fully pressed)
+    //
+    optional double pedal_position_clutch = 3;
+}
+
+//
+// \brief A description of the steering wheel.
+//
+message VehicleSteeringWheel
+{
+    // Angle of the steering wheel. 
+    // Zero means the steering wheel is in its center position, a positive value
+    // means the steering wheel is turned to the left and a negative value
+    // means the steering wheel is turned to the right of the center position.
+    //
+    // Unit: rad
+    //
+    optional double angle = 1;
+
+    // Angular speed of the steering wheel. 
+    // Zero means the steering wheel stays in its position, a positive value
+    // means the steering wheel is turned to the left and a negative value
+    // means the steering wheel is turned to the right.
+    //
+    // Unit: rad/s
+    //
```

Comment:
range

Reply:
I think it should stay unlimited as it is depends on the vehicle, where the max. values vary.